pkg/v1/providers/config_default.yaml

```diff
@@ -582,7 +582,7 @@ AVI_MANAGEMENT_CLUSTER_CONTROL_PLANE_VIP_NETWORK_NAME:
 AVI_MANAGEMENT_CLUSTER_CONTROL_PLANE_VIP_NETWORK_CIDR:
 
 AVI_CA_DATA_B64: ""
-AVI_LABELS: ""
```

Comment:
If we change this, will it cause anything changed here:https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/providers/ytt/02_addons/avi/ako_operator_secret.yaml#L9?